aips/passkeys.md

```diff
@@ -0,0 +1,412 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Passkeys
+author: hariria
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: <Draft | Last Call | Accepted | Final | Rejected>
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedback and reviews>
+type: Standard (Core/Framework)
+created: <12/14/2023>
+updated (*optional): <mm/dd/yyyy>
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-X - Passkey accounts
+
+## Summary
+
+This AIP proposes the first WebAuthn Authenticator for Aptos, enabling users to utilize passkeys and other WebAuthn credentials for on-chain accounts and transaction signing.
+
+[Passkeys](https://fidoalliance.org/passkeys/) are designed to replace passwords as a phishing resistant, faster, and more secure form of user authentication. When a user registers a passkey, a new [public key credential](https://www.w3.org/TR/webauthn-3/#public-key-credential) is created on their device's [authenticator](https://www.w3.org/TR/webauthn-3/#authenticator) - which holds and manages the credentials for that user. As of February 2023, on iOS 16+, Android 9+, and MacOS 13+, the passkey can then securely sync with their device's cloud provider (iCloud or Google Password Manager, for example) using end-to-end encryption to ensure that the passkey is recoverable in the event that the device is lost. When signing in with the passkey, the user is prompted to authenticate - often with a biometric based [authorization gesture](https://www.w3.org/TR/webauthn-3/#authorization-gesture) like Face ID or Touch ID - and provides a response that proves ownership of the passkey via a digital signature.
+
+Passkeys use the [WebAuthn specification](https://www.w3.org/TR/webauthn-3/), a standard co-created by the [FIDO alliance](https://fidoalliance.org/) and the [World Wide Web Consortium (W3C)](https://www.w3.org/), for passkey registration and authentication. We are choosing to use NIST P256 (`secp256r1`) as the first enabled WebAuthn signature scheme because of its broad support across most modern operating systems. Once complete, this will enable users to sign and submit transactions with any compatible WebAuthn credential on Aptos, including multi-device credentials registered on iOS, MacOS, and Android devices, as well as single-device, hardware-bound credentials on devices like Yubikeys.
```

Comment:
Probably too much content, let's try to rewrite this purely as a summary for application to Aptos: 

```suggestion
On Aptos, passkeys authenticate transactions by using a [WebAuthn](https://www.w3.org/TR/webauthn-3/)-specific `AccountAuthenticator`. `AccountAuthenticators` are mentioned in [AIP-55](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-55.md). Aptos currently supports NIST P256 (`secp256r1`) as the only WebAuthn signature scheme because of its broad support across most modern operating systems. This functionality allows a users to sign and submit transactions with any compatible WebAuthn credential on Aptos, including multi-device credentials registered on iOS, MacOS, and Android devices, as well as single-device, hardware-bound credentials on devices like Yubikeys.
```

Reply:
Will word-smith and add